### PR TITLE
Adds alternate bowman headsets and an actual headset for the Bodyguard as well as the Hailing frequency to Nerva

### PR DESCRIPTION
**Pull request by ErrorsWindows**
My First Pull Request:tm:

tl;dr title

Unfortunately this is not TG and bowmans don't actually impart any kind of hearing protection onto the wearer so they only exist here for aesthetic
Also the Bodyguard was just using the head of personnel's headset this whole time so I gave them their own (say goodbye to the Service frequency)
Also also the Hailing frequency is a thing that exists so I added the functionality to the Nerva bridge roles as well
Also also also the Bodyguard has a unique locker type that isn't just a reskinned blueshield locker but this has basically no impact on gameplay except for code

🆑 
* add: Alternate bowman headsets for (most) job lockers
* add: An actual bodyguard headset (instead of the Head of Personnel's headset they were using before)
* tweak: Nerva bridge roles (except the Bodyguard and Senior Scientist) get access to the Hailing frequency (instead of just the CMO, CE, and Bodyguard strangely enough)

🆑

### CODE REVIEW

File: maps/nerva/obj/nerva_items.dm [hunk 1]
```diff
@@ -16,55 +16,125 @@
...
+/obj/item/device/radio/headset/heads/nerva_senior/alt
+	name = "senior scientist bowman headset"
+	desc = "The bigger headset of the ICS Nerva's Senior Scientist."
+	icon_state = "nt_headset_alt"
+	item_state = "nt_headset_alt"
+	ks1type = /obj/item/device/encryptionkey/nerva/senior
+
+/obj/item/device/radio/headset/nerva_guard
+	name = "bodyguard's bowman radio headset"
+	desc = "This is used by the captain's escort."
```
Comment (CaraS212) [inline maps/nerva/obj/nerva_items.dm:126]: hot damn that is some phrasing
keep it and make the captain and guard suffer

File: maps/nerva/obj/nerva_items.dm [hunk 2]
```diff
@@ -16,55 +16,125 @@
...
+	ks1type = /obj/item/device/encryptionkey/nerva/cap
+
 /obj/item/device/radio/headset/heads/nerva_cap
 	name = "captain's headset"
 	desc = "The headset of the ICS Nerva's captain."
 	icon_state = "com_headset"
 	item_state = "headset"
 	ks1type = /obj/item/device/encryptionkey/nerva/cap
 
+/obj/item/device/radio/headset/heads/nerva_cap/alt
```
Comment (Glloyd) [inline maps/nerva/obj/nerva_items.dm:61]: you haven't added this to the nerva captain's locker, which is in nerva_closets.dm at the top of the file.

**Review by Glloyd — CHANGES_REQUESTED** [review]
just a couple little issues that will need to be addressed

File: code/modules/urist/structures/closets.dm [hunk 3]
```diff
@@ -60,6 +60,8 @@
 	new /obj/item/grenade/flashbang(src)
 	new /obj/item/melee/baton/loaded(src)
 	new /obj/item/gun/energy/taser(src)
+	new /obj/item/device/radio/headset/nerva_guard(src)
+	new /obj/item/device/radio/headset/nerva_guard/alt(src)
```
[the remark this answers is not in the document]
Comment (ErrorsWindows) [reply code/modules/urist/structures/closets.dm]: ok i made a new bodyguard's locker that isn't just a reskinned blueshield locker

File: maps/nerva/obj/nerva_items.dm (hunk 2 above)
Comment (ErrorsWindows) [reply maps/nerva/obj/nerva_items.dm:61]: whoops

**Comment by Glloyd** [comment]
You had the locker right, it was just you misspelled the closet_appearance var

**Review by Glloyd — APPROVED** [review]
🎉 congrats on your first merged PR
